testnet/stacks-node/src/config.rs

```diff
@@ -1302,9 +1382,34 @@ pub struct BurnchainConfigFile {
     pub sunset_end: Option<u32>,
     pub wallet_name: Option<String>,
     pub ast_precheck_size_height: Option<u64>,
+    pub affirmation_overrides: Option<Vec<AffirmationOverride>>,
 }
 
 impl BurnchainConfigFile {
+    /// Add affirmation overrides required to sync Xenon
```

Comment:
Maybe add a comment to explain to the reader how these affirmation maps can be independently recreated?

Reply:
Good idea. I have no idea where these came from XD I just got them off @wileyj .

Reply:
@kantai do you have any insights?